src/Build/Definition/Project.cs

```diff
@@ -2686,37 +2686,37 @@ ProvenanceResult SingleItemSpecProvenance(string itemSpec, IElementLocation elem
                     }
 
                     var matchOccurrences = ItemMatchesInItemSpecString(itemToMatch, itemSpec, elementLocation, itemElement.ContainingProject.DirectoryPath, _data.Expander, out Provenance provenance);
-                    Tuple<Provenance, int> result = matchOccurrences > 0 ? Tuple.Create(provenance, matchOccurrences) : null;
```

Comment:
Grrr...this was my part of my pet project for when I don't know what I should be working on. (It is related to an SLA bug I was assigned.)

I cleaned it up more. See [here](https://github.com/Forgind/msbuild/blob/get-item-provenance/src/Build/Definition/Project.cs#L2678-L2736). I don't think you broke anything, though.

Reply:
let me know if you'd like me to take this commit out to not cause you conflicts

Reply:
Don't worry about it. I don't have a timeline for when my commits might turn into a PR, and this is definitely a huge improvement over what was there.